### ISSUE TITLE
Critical (but cryptic error) when using packer > 1.3.1 on linux platforms

### ISSUE THREAD

**Issue by obourdon** [hashicorp/docker-hub-images#86]
Before I updated my packer build pipeline to any version > 1.3.1, everything was working like a charm. Now I get the following cryptic error:
```
Status: Downloaded newer image for hashicorp/packer:1.4.3
amazon-ebs output will be in this color.

Build 'amazon-ebs' errored: error validating regions: UnauthorizedOperation: You are not authorized to perform this operation.
	status code: 403, request id: 9126ba8a-3f99-4aec-a23a-155a46f5b385

==> Some builds didn't complete successfully and had errors:
--> amazon-ebs: error validating regions: UnauthorizedOperation: You are not authorized to perform this operation.
	status code: 403, request id: 9126ba8a-3f99-4aec-a23a-155a46f5b385

==> Builds finished but no artifacts were created.
make: *** [Makefile:49: build] Error 1
script returned exit code 2
```

Adding PACKER_LOG=1 does not give much more information on the subject.

After googling a lot and putting some debug traces in my own packer executable I found out what was wrong: the alpine image is missing CA certificates :-(

Note that this seems to be working on MacOSX platforms without any change. I did not check either on other platforms

**Comment by obourdon** [comment]
Forgot to mention that my test linux systems were Ubuntu 16.04 and Debian 9

**Comment by SwampDragons** [comment]
This PR is merged now, and the dockerfile for v1.5.2 contains these changes. Sorry for the long delay.
